edxval/utils.py

```diff
@@ -0,0 +1,27 @@
+"""
+Util methods to be used in api and models.
+"""
+
+from django.conf import settings
+from django.core.files.storage import get_storage_class
+
+
+def video_image_path(video_image_instance, filename):  # pylint:disable=unused-argument
+    """
+    Returns video image path.
+    """
+    return '{}{}'.format(settings.VIDEO_IMAGE_SETTINGS.get('DIRECTORY_PREFIX', ''), filename)
```

Comment:
I would use `u` along with format. This has beaten us many time at different places.